.jscsrc

```diff
@@ -0,0 +1,5 @@
+{
+  "preset": "google",
+  "maximumLineLength": 120,
+  "excludeFiles": ["node_modules/**", "reports/**", "dist/js/bundle.js"]
```

Comment:
`node_modules` are ignored by default, `reports` isn't generated anymore, `dist` has been renamed to `public`